lib/MiLight/MiLightClient.cpp

```diff
@@ -267,16 +267,23 @@ void MiLightClient::update(const JsonObject& request) {
     uint8_t r = color["r"];
     uint8_t g = color["g"];
     uint8_t b = color["b"];
+    //If close to white
+    if( r > 256 - RGB_BOUNDARY && g > 256 - RGB_BOUNDARY && b > 256 - RGB_BOUNDARY)
```

Comment:
Would you mind updating the styling? e.g.:

```
if (...) {
  ...
} else {
  ...
}
```

to be consistent with the rest of the project.